lib/goth/application.ex

```diff
@@ -6,7 +6,7 @@ defmodule Goth.Application do
   def start(_type, _args) do
     children = [
       {Registry, keys: :unique, name: Goth.Registry},
-      {Finch, name: Goth.Finch, pools: %{default: [protocol: :http1]}},
+      {Finch, name: Goth.Finch, pools: %{default: [protocols: [:http1]]}},
```

Comment:
http1 is the default, we can just drop the :pools option and it will work across all Finch versions. I don't know if there was a particular reason to use HTTP/1. If anything we could probably assume Google uses HTTP/2 but I wouldn't switch to :http2.

Could you drop :pools options here instead?

Reply:
Sure thing, I noticed that we're setting the default explicitly here, and since I don't know the reasoning behind that, didn't want to change it right away, as all I'm trying to do is fix a runtime warning. But if it is ok to drop it, absolutely no problem, done :+1: